README.md

```diff
@@ -1,5 +1,5 @@
 [![Puppet Forge](http://img.shields.io/puppetforge/v/katello/pulp.svg)](https://forge.puppetlabs.com/katello/pulp)
-[![Build Status](https://travis-ci.org/Katello/puppet-pulp.svg?branch=master)](https://travis-ci.org/Katello/puppet-pulp)
+[![Build Status](https://travis.ci.org/theforeman/puppet-pulp.svg?branch=master)](https://travis-ci.org/Katello/puppet-pulp)
```

Comment:
travis.ci.org? Fixed in https://github.com/theforeman/puppet-pulp/pull/290

Reply:
Looks like I messed up my sed replacement. Thanks for pointing it out, will fix the other module as well.